src/snowflake/snowpark/functions.py

```diff
@@ -1419,7 +1520,21 @@ def pow(
 
 def round(e: ColumnOrName, scale: Union[ColumnOrName, int, float] = 0) -> Column:
     """Returns values from the specified column rounded to the nearest equal or
-    smaller integer."""
+    smaller integer.
```

Comment:
This is not true, 1.5 is rounded to 2 by default, the [Snowflake round function](https://docs.snowflake.com/en/sql-reference/functions/round) has a third parameter `mode` that can be either [half away from zero](https://en.wikipedia.org/wiki/Rounding#Rounding_half_away_from_zero) or [half to even](https://en.wikipedia.org/wiki/Rounding#Rounding_half_to_even). The default is rounding half away from zero.

Reply:
good finding, I removed `nearest equal or smaller integer.` description, I think just using `rounded` should be good